index.bs

```diff
@@ -2123,43 +2123,27 @@ The following abstract operations operate on {{ReadableStream}} instances at a h
 </div>
 
 <div algorithm>
- <dfn abstract-op lt="ReadableStreamFromIterable" id="readable-stream-from-iterable">
+ <dfn abstract-op export lt="ReadableStreamFromIterable" id="readable-stream-from-iterable">
```

Comment:
Why export, btw?

Reply:
Will be needed to resolve https://github.com/whatwg/fetch/issues/1291, which is my next project :)

Reply:
In that case you would want a separate op in section 9, this one is in section 4 and not supposed to be directly used by other specs.

Reply:
Ok, will revert this here